Automattic/jetpack
# P2: hide Jetpack menu for all P2

#### Changes proposed in this Pull Request:
As per discussion `p1659050418704389-slack-p2`, I'm proposing a PR to remove Jetpack menu for all P2 (hubs and spaces) from Calypso admin menu, where the logic was only show Jetpack menu for paid P2s, the original PR is https://github.com/Automattic/jetpack/pull/22148.

#### Other information:


#### Jetpack product discussion
p1659050418704389-slack-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Apply patch to your sandbox
- Ensure for all P2 sites, Jetpack menu is hidden
